src/components/place-card-detailed/place-card-detailed.jsx

```diff
@@ -0,0 +1,199 @@
+import React from "react";
+import PropTypes from "prop-types";
+import OffersList from "../offers-list/offers-list";
+import ReviewForm from "../review-form/review-form";
+import {propTypeOffer, propTypeReview} from "../../check-prop-types";
+
+
+const PlaceCardDetailed = (props) => {
+  const {offers} = props;
+  const {reviews} = props;
+  const {id} = props;
+  const {history} = props;
+
+  const el = offers.filter((offer) => offer.id === parseInt(id, 10))[0];
+  const favOffers = offers.filter((offer) => offer !== el).filter((item) => item.city === el.city).slice(0, 3);
+  const options = {
+    articleClassName: `near-places__card`,
+    wrapperClassName: `near-places__image-wrapper`,
+    infoClassName: `near-places__card-info`,
+    image: {
+      imgWidth: 260,
+      imgHeight: 200,
+    }
+  };
+
+  return (
+    <div className="page">
+      <header className="header">
+        <div className="container">
+          <div className="header__wrapper">
+            <div className="header__left">
+              <a className="header__logo-link" href="main.html">
+                <img className="header__logo" src="img/logo.svg" alt="6 cities logo" width="81" height="41"/>
+              </a>
+            </div>
+            <nav className="header__nav">
+              <ul className="header__nav-list">
+                <li className="header__nav-item user">
+                  <a className="header__nav-link header__nav-link--profile" href="#">
+                    <div className="header__avatar-wrapper user__avatar-wrapper">
+                    </div>
+                    <span className="header__login">Sign in</span>
+                  </a>
+                </li>
+              </ul>
+            </nav>
+          </div>
+        </div>
+      </header>
+
+      <main className="page__main page__main--property">
+        {
+          <section key={el.title} className="property">
+            <div className="property__gallery-container container">
+              <div className="property__gallery">
+                {el.pictures.map((picture, i) => (
+                  <div className="property__image-wrapper" key={i * Math.random()}>
+                    <img className="property__image" src={picture} alt="Photo studio"/>
+                  </div>
+                ))}
+              </div>
+            </div>
+            <div className="property__container container">
+              <div className="property__wrapper">
+                {
+                  el.premium ?
+                    <div className="property__mark">
+                      <span>Premium</span>
+                    </div>
+                    : ``
```

Comment:
можно записать короче: 
```suggestion
                  el.premium &&
                    <div className="property__mark">
                      <span>Premium</span>
                    </div>
```